first_hour_on_vwb/widget_utils.py

```diff
@@ -0,0 +1,92 @@
+import json
+import ipywidgets as widgets
+
+input_style = {
+    'description_width':'initial'
+}
+
+input_layout = widgets.Layout(
+    width='75%',
+    align_items='flex-start'
+)
+
+output_style = {
+    'description_width':'initial',
+}
+
+vbox_layout = widgets.Layout(
+    border='solid 2px',
+    padding='10px',
+    display='flex',
+    flex_flow='column',
+    align_items='flex-start',
+    width='75%'
+)
+
+def users_table(stdout):
```

Comment:
nit: Looks like you're parsing an arbitrary JSON string, not  just those from stdout.  Consider renaming the argument?

Reply:
done!